TransferMethod/Tests/Responses/BankAccountIndividualUpdateResposne.json

```diff
@@ -0,0 +1,7 @@
+{
+  "userToken": "usr-c4292f1a-866f-4310-a289-b916853939de",
+  "token": "trm-56b976c5-26b2-42fa-87cf-14b3366673c6",
+  "status": "ACTIVATED",
+  "createdOn": "2020-12-11T13:34:35.762Z"
+}
```

Comment:
use the mock response from the Core SDK ticket that Rita provided

Reply:
Rita provided response for getting field details only. I added response for updating transfer method.